examples/animatediff/README.md

```diff
@@ -44,25 +44,121 @@ Instruction on ffmpeg and decord install on EulerOS:
 First, download the torch pretrained weights referring to [torch animatediff checkpoints](https://github.com/guoyww/AnimateDiff/blob/main/__assets__/docs/animatediff.md#download-base-t2i--motion-module-checkpoints).
 
 - Convert SD dreambooth model
+
+To download ToonYou-Beta3 dreambooth model, please refer to this [civitai website](https://civitai.com/models/30240?modelVersionId=78775), or use the following command:
+```
+wget https://civitai.com/api/download/models/78755 -P models/torch_ckpts/ --content-disposition --no-check-certificate
```

Comment:
`--no-check-certificate` is not nessesary

Reply:
After testing this command on 910B, I found `--no-check-certificate` is needed.

Reply:
`--no-check-certificate` depends on the machine environment and varies from user to user.

Reply:
Thanks. Then I will keep it.